docs/30_getting-started-guide/index.md

```diff
@@ -3,12 +3,27 @@ content_title: "Getting Started Guide"
 link_text: "Getting Started Guide"
 ---
 
-The EOSIO _Getting Started Guide_ provides guided instructions on how to set up your local development environment. These instructions installs the EOSIO software components and other related dependencies for the EOSIO platform. This setup is an efficient and convenient method to onboard the EOSIO development ecosystem. Subsequently, you become acquainted with the smart contracts development workflow in a local environment.
+The purpose of the _EOSIO Getting Started Guide_ is to provide guided instructions to set up your local EOSIO development environment and use the environment to build and deploy a basic **_Hello World_** Smart Contract. The _Getting Started Guide_ high-level workflow is illustrated in the following diagram: 
+
+![EOSIO Getting Started Workflow](../eosio-gsg-workflow.png)
```

Comment:
This link is broken. Should be:
!\[EOSIO Getting Started Workflow\](eosio-gsg-workflow.png)